apps/gallery/js/startup.js

```diff
@@ -146,8 +146,9 @@ navigator.mozL10n.once(function startup() {
       }
 
       // Hide the scanning indicator
-      $('progress').classList.add('hidden');
+      // setTimeout() is to workaround Bug 1166500
       $('throbber').classList.remove('throb');
+      setTimeout(function() { $('progress').classList.add('hidden'); }, 100);
```

Comment:
Originally I did it without the 100ms, and that usually worked, but at least once it did not work. I don't undertstand why that happened, but I figured it was safer to just use something longer than an animation frame to be sure that the animation has actually stopped before hiding the container.